source/server-discovery-and-monitoring/tests/rs/discover_arbiters.yml

```diff
@@ -1,6 +1,6 @@
 description: "Discover arbiters"
```

Comment:
I think this should be renamed to "Discover arbiters with directConnection URI option" just so that the difference between this and the duplicated test is more explicit. I don't think renaming the test file is necessary, but since the description is used to identify the tests in evergreen (at least this is true for PyMongo), I think there is value in making it explicit w.r.t. what is being tested.

Reply:
If folks agree, this would apply to descriptions from all tests that now use the `directConnection` option.

Reply:
Done.